### DISCUSSION TITLE
You need a stronger password

### DISCUSSION THREAD

**Discussion by matfeinikolaev** [supabase/supabase#2694] [Questions] [answered]
When creating a new project, the form doesn't accept any password for the Database password, saying "You need a stronger password". The "Create new project" button is hence disabled. I have tried all variants of passwords, with uppercase, lowercase letters, digits, symbols, in strings of up to 2048 characters. Nothing gets accepted.
![image](https://user-images.githubusercontent.com/45239725/128061605-769fc7a9-bdc2-45e0-af97-d9ce9886e415.png)

**Comment by inian** [comment, marked as answer]
Lets continue the discussion in the issue you created. https://github.com/supabase/supabase/issues/2696

Please open only a issue _or_ a discussion, else it doubles the work for triaging them
